README.md

```diff
@@ -56,29 +56,27 @@ ENV HOSTNAME my-docker-instance-123
 
 ```
 
-  or environment settings can be passed at container launch time:
-
-- Use the `-e` option with `docker run` as in
+Alternatively, environment settings can be passed at the container launch time. Use the `-e` option with `docker run`, for example:
 
     ```bash
     docker run --name mynginx1 -e ENV_CONTROLLER_API_KEY=1234567890 -e ENV_CONTROLLER_INSTANCE_NAME=my-instance-123 -d nginx-agent
     ```
 
 ### 1.3. Current Limitations
 
-The following list summarizes existing limitations of monitoring containers with NGINX Controller:
+The following list summarizes the existing limitations of monitoring containers with NGINX Controller:
 
-- The agent can only monitor NGINX from inside the container. It is not currently possible to run the agent in a separate container and monitor the neighboring containers running NGINX.
+- The Controller Agent can only monitor NGINX from inside the container. It is not currently possible to run the Controller Agent in a separate container and monitor the neighboring containers running NGINX.
 
-## 2. How to Build and Run a Controller enabled NGINX image
+## 2. How to Build and Run an NGINX Controller-enabled NGINX image
 
-### 2.1. Building a Controller-enabled image with NGINX
+### 2.1. Building an NGINX Controller-enabled image with NGINX
 
-(**Note**: If you are new to Docker, [here's](https://docs.docker.com/engine/installation/) how to install Docker Engine on various OS.)
+(**Note**: If you are new to Docker, refer to the documentation on [how to install Docker Engine on various operating systems](https://docs.docker.com/engine/installation/).)
 
-**Note** Before proceeding, you must: install NGINX Controller, download your certificate and key for NGINX Plus, obtain the api key for your NGINX Contoller instance.
+**Note** Before proceeding, you must install NGINX Controller, download your certificate and key for NGINX Plus, and obtain the API key for your NGINX Controller instance.
```

Comment:
We should add mention that the nginx repo cert and key should be called nginx-repo.crt and nginx-repo.key as those names are required.

Reply:
Hi @luk90 -- I updated the text with the file names and included links to the supporting docs. Take a look and let me know if it's okay.

**Note**: The doc that I linked to about how to download the N+ cert and key will be published when Controller 3.11 goes live, so we should wait to update this README until then; otherwise, the link won't work. If you need to publish the README sooner, we can remove the link and then add it back when the doc is available.